engine/cluster/src/main/java/org/opennms/oce/engine/cluster/ClusterEngine.java

```diff
@@ -203,6 +203,17 @@ public void destroy() {
         // no-op
     }
 
+    @Override
+    public void deleteSituation(String situationId) {
+        LOG.trace("Deleting situation references for situation Id {}", situationId);
+        situationsById.remove(situationId);
+        Set<String> alarmIdsToDelete = alarmIdToSituationMap.entrySet().stream()
```

Comment:
Can we query the retrieve the ids from the situation itself, rather than iterating over the whole map?

Reply:
Yep, didn't notice that it had the Ids.